src/docs/user/references/GenericInformation.adoc

```diff
@@ -22,10 +22,63 @@ For example:
 </task>
 ----
 
-=====  Icon management variables
+Below are several examples of Generic information:
 
-There are specific variables that are dedicated to icon management.
-The icon of a workflow is specified inside Workflow Generic Information using the keyword `workflow.icon`.
-The icon of a task is specified inside Task Generic Information using the keyword `task.icon`.
+==== START_AT
+
+The `START_AT` Generic Information can be used to delay a job or task execution at a specified date/time.
+Its value should be https://en.wikipedia.org/wiki/ISO_8601[ISO 8601^] compliant. See <<_cron_tasks>> for more details.
+
+Examples:
+
+ * `START_AT = "2020-06-20T18:00:00"` will delay the job execution until 20th June 2020 at 6pm GMT.
+ * `START_AT = "2020-06-20T18:00:00+02:00"` will delay the job execution until 20th June 2020 at 6pm GMT+02:00.
+
+`START_AT` can be defined at *job-level* (delay the execution of the whole job) or at  *task-level* (delay the execution of a single task).
+
+
+==== REMOVE_DELAY
+
+The `REMOVE_DELAY` Generic Information can be used to control when a job is removed from the scheduler database after its termination.
+
+The <<../admin/ProActiveAdminGuide.adoc#_housekeeping,housekeeping mechanism>> must be configured to allow usage of `REMOVE_DELAY`.
+
+`REMOVE_DELAY` overrides the global `pa.scheduler.core.automaticremovejobdelay` setting for a particular job. It allows a job to be removed either *before* or *after* the globally-configured delay.
```

Comment:
`the globally-configured delay.` is confusing. I am not sure to understand how it is related to the `pa.scheduler.core.automaticremovejobdelay` configuration key.